server/zally-server/src/main/java/org/zalando/zally/apireview/Score.kt

```diff
@@ -0,0 +1,56 @@
+package org.zalando.zally.apireview
+
+import org.zalando.zally.rule.api.Severity
+import java.math.BigDecimal
+import java.math.RoundingMode
+import java.util.EnumMap
+import kotlin.math.min
+
+class Score {
+    companion object {
```

Comment:
why static / as companion?

Reply:
it's only a function, not an object with a value -> do you suggest to rewrite it to oo-style. would be possible